docker-compose.yml

```diff
@@ -22,6 +22,10 @@ services:
       target: development
     environment:
       # Be sure to add environment variables to config/options.yml
+      - APP_URL_PROTOCOL
+      - APP_URL_HOST
+      - APP_URL_PORT
+      - VALIDATE_ENV_VARS
```

Comment:
I was going to say that we need to search the code base for `RUN_SOLID_QUEUE_IN_PUMA` to find all the other places we're handling env config for our production environments, but we won't need to do that if we can make all the keys successfully be optional.